Form-Controls/index.html

```diff
@@ -4,24 +4,90 @@
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <title>My form exercise</title>
-  <meta name="description" content="">
+  <meta name="description" content="A form for ordering T-Shirt in three color and 6 size
+  Author: Delnia Alipour">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="stylesheet" href="styles.css">
 </head>
 <body>
   <header>
     <h1>Product Pick</h1>
   </header>
-  <main>
-  <form>
     <!-- write your html here-->
     <!-- try writing out the requirements first-->
-  </form>
+    <main class="store-content">
+      <section class="form-section">
+         
+          <form class="form-store" method="">
+             <fieldset class="fieldset-store">
+                <div class="full-name">
+                 <div class="first-name">
+                     <label for="firstname">First name</label>
+                     <input class="form-input" type="text" name="fisrtname" id="firstname" pattern="[a-zA-Z]+.{2,}" placeholder="Enter your first name" required>
```

Comment:
```suggestion
                     <input class="form-input" type="text" name="firstname" id="firstname" pattern="[a-zA-Z]+.{2,}" placeholder="Enter your first name" required>
```
Typos cause such problems! ;) There are spellcheckers you can install in VSCode.